tests/Sniffs/Commenting/Fixtures/RequiredDocBlockSniff/correct/Variable.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+/**
+ * Valid Doc Block.
+ */
+class ClassWithDocBlock {
+    protected string $foo = 'BAR';
```

Comment:
This does not make any sense for the ADDED ERROR with the message self::MESSAGE_MISSING_DOC_BLOCK_VAR ... There should be a new error test and not an added correct fixture. And event then: This feature should not remove any doc blocks!